NitroxLauncher/Pages/LaunchGamePage.xaml

```diff
@@ -1,132 +1,119 @@
-
-            </StackPanel>
-
-        </Grid>
-        <Image Stretch="Uniform" Source="{StaticResource SubnauticaMainImage}" Height="48" VerticalAlignment="Center" Margin="193,161,192,186" Grid.Row="1" Grid.Column="1" HorizontalAlignment="Center" />
-    </Grid>
+﻿<!-- TODO: Responsive width when resizing the image -->
+<local:PageBase x:Class="NitroxLauncher.Pages.LaunchGamePage"
+                xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                xmlns:local="clr-namespace:NitroxLauncher.Models"
+                xmlns:converters="clr-namespace:NitroxLauncher.Models.Converters"
+                DataContext="{Binding RelativeSource={RelativeSource Self}}"
+                mc:Ignorable="d" d:DesignWidth="766"
+                Title="Launch Game" Height="1020">
+
+    <local:PageBase.Resources>
+        <BitmapImage x:Key="PlayGameImage" UriSource="pack://application:,,,/Images/PlayGameImage.png" />
+        <BitmapImage x:Key="SubnauticaMainImage" UriSource="pack://application:,,,/Images/SubnauticaMain.png" />
+        <BitmapImage x:Key="NitroxGallery" UriSource="pack://application:,,,/Images/NitroxGallery.png" />
+    </local:PageBase.Resources>
+
+    <Grid >
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="22*"/>
+            <ColumnDefinition Width="675*"/>
+            <ColumnDefinition Width="22*"/>
+        </Grid.ColumnDefinitions>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="0" />
+            <RowDefinition Height="475" />
+            <RowDefinition Height="539" />
+        </Grid.RowDefinitions>
+
+        <Image Stretch="UniformToFill" Source="{StaticResource PlayGameImage}" Margin="0,0,0,22" Grid.Row="1" Grid.ColumnSpan="3" />
+
+        <Grid Grid.Row="1" VerticalAlignment="Top" Height="644" Grid.Column="1" RenderTransformOrigin="0.503,0.447" Margin="0,370,0,0" Grid.RowSpan="2">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="123*"/>
+                <RowDefinition Height="84*"/>
+                <RowDefinition Height="115*"/>
+            </Grid.RowDefinitions>
+
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="0*"/>
+                <ColumnDefinition Width="174*"/>
+                <ColumnDefinition Width="203*"/>
+                <ColumnDefinition Width="342*"/>
+                <ColumnDefinition Width="0*"/>
+            </Grid.ColumnDefinitions>
+
+            <TextBlock FontSize="14" Foreground="White" TextWrapping="Wrap" Grid.Column="1" Margin="0,112,20,0" Height="87" VerticalAlignment="Top" Grid.ColumnSpan="2">
+                <Run Text="Nitrox is an open-source, multiplayer modification for the game Subnautica."/>
+                <Run Text=" "/>
```

Comment:
Adding this space seems to add a double space within the paragraph, is this intended?
![image](https://user-images.githubusercontent.com/8798074/141024322-b3dbaa74-31bc-4b18-831e-d2ccc88a789b.png)